GitCommands/Git/GitModule.cs

```diff
@@ -980,15 +980,15 @@ public GitRevision GetRevision(ObjectId? objectId = null, bool shortFormat = fal
             revision.HasNotes = !shortFormat;
             if (shortFormat)
             {
-                revision.Subject = ReEncodeCommitMessage(lines[10], revision.MessageEncoding);
+                revision.Subject = ReEncodeCommitMessage(lines[10], revision.MessageEncoding) ?? "";
```

Comment:
In a few places, null is substituted for a default value. In this case, the empty string.